packages/expo-local-authentication/src/LocalAuthentication.ts

```diff
@@ -52,9 +57,19 @@ export async function authenticateAsync(
 
     const result = await ExpoLocalAuthentication.authenticateAsync(options);
 
+    if (Constants.appOwnership === 'expo') {
+      const hasFaceID = (await supportedAuthenticationTypesAsync()).includes(
+        AuthenticationType.FACIAL_RECOGNITION
+      );
+      if (hasFaceID) {
+        result.warning =
+          'FaceID is not available in the Expo Client. You can use it in a standalone Expo app by providing `ios.infoPlist.NSFaceIDUsageDescription` in your app.json file.';
```

Comment:
I think it might be better and more clear if we do this on the native side since we already handle warnings from native 🤔

Reply:
I second that. By adding this to the native side we could not ship any of this code to bare workflow apps, which is something of a goal. (We would subclass LocalAuthentication native module in ExpoKit and check then be able to know if we're inside Expo client or not.)

Reply:
Looks like this is already done now in `EXScopedLocalAuthentication`, and I've tested on Expo Client SDK 35 and the correct error message is logged in the client (`FaceID is not available in Expo Client....`), so maybe this can be closed?